ui/perfherder/alerts/AlertTable.jsx

```diff
@@ -248,6 +248,17 @@ export default class AlertTable extends React.Component {
                   </tr>
                 </thead>
                 <tbody>
+                  <tr>
+                    <th>&nbsp;</th>
+                    <th>&nbsp;</th>
+                    <th>Alert Name</th>
+                    <th>&nbsp;</th>
+                    <th>&nbsp;</th>
+                    <th>&nbsp;</th>
+                    <th>Detail Hint</th>
+                    <th>Magnitude</th>
```

Comment:
These titles aren't quite right. If you look at the values being used in the table and tooltip text, they will give you a hint. We should also add titles for all columns.

Reply:
Please also give this row the same thin grey border as the rest of the table.

Reply:
Yes, those are fine except "Alert name" column text should be "Test and platform".